Add more questions for ML Chapter 4 (#6)
Qala-Dev/discussion-questions: mastering-lightning/README.md

    Do you think it is beneficial to have more or less of these in a non-consensus network?
1. Mastering Lightning demonstrates setting up nodes in docker for testing.
    Is it safe to then use these docker nodes in production later?

> 1. Do you think all companies developing on the lightning network should be required to open-source their Lightning apps?

Reviewer: ```suggestion
1. Do you think all companies developing on the Lightning Network should be required to open-source their Lightning apps?
```

> 1. Do the different implementations of Lightning Network help or hurt Lightning Network and why?

Reviewer: ```suggestion
1. Does it help or hurt the network to have different Lightning Network node implementations, and why?
```

> 1. Are there ways to check the integrity of all the libraries used in this demo?

Reviewer: ```suggestion
1. What are the ways to check the integrity of all the software used in this demo? Which software have you verified it for?
```
